db/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM mariadb:10.2
+
+ENV MYSQL_USER=beaker \
+      MYSQL_PASSWORD=beaker \
+      MYSQL_ALLOW_EMPTY_PASSWORD=yes
```

Comment:
Do we need this option?

Reply:
Yes - it allows us to have an empty root password for the mariadb server. 
We could set a root password but there's no need to do so imho.